model/flow/identifier.go

```diff
@@ -121,6 +121,14 @@ func MakeID(entity interface{}) Identifier {
 	return id
 }
 
+func MakeIDFromBytes(data *[]byte) Identifier {
```

Comment:
Why do we have a pointer to the slice here? I would consider it better Go-style to use the slice directly:

```suggestion
func MakeIDFromBytes(data []byte) Identifier {
```